src/animation/AnimationClip.js

```diff
@@ -16,11 +17,12 @@ import { MathUtils } from '../math/MathUtils.js';
  * @author David Sarno / http://lighthaus.us/
  */
 
-function AnimationClip( name, duration, tracks ) {
+function AnimationClip( name, duration, tracks, blendMode ) {
 
 	this.name = name;
 	this.tracks = tracks;
 	this.duration = ( duration !== undefined ) ? duration : - 1;
+	this.blendMode = blendMode || NormalAnimationBlendMode;
```

Comment:
We avoid this pattern because it assumes `blendMode` is non-zero.

It is best to use the same pattern as is used for `duration`.